bot/exts/utils/snekbox/_eval.py

```diff
@@ -92,10 +92,17 @@ def get_message(self, job: EvalJob) -> tuple[str, str]:
         # Add error message for failed attachments
         if self.failed_files:
             failed_files = f"({', '.join(self.failed_files)})"
```

Comment:
- Moved message to be below output
- Changed wording a bit, now shows count of failed files.
- Truncation on whole file names, either exceeding 85 characters, or 5 files.
<img width="764" alt="image" src="https://user-images.githubusercontent.com/13956642/208238523-972b4684-db8d-48e5-a096-3d57a885af8e.png">

- If a single file exceeds 85 characters, only displays `(...)`
<img width="764" alt="image" src="https://user-images.githubusercontent.com/13956642/208238610-6944345c-2e6c-4e9b-9cf3-ca40aca36a4a.jpg">

Reply:
Instead of just showing `...` when it's too long, can it truncate the file name e.g. `1 file (reallylongfi...)`?

Also, I'm okay with the current logic of 85 chars or 5 files, but I would have just had a character limit and truncated the last filename in the list. Let's say there are 10 files, but they are under 85 chars in total. It's free real estate being given up I think.

Reply:
Actually this reminded me, we need to do some sanitizing of file paths before attempting to upload to discord, since they only allow ascii-alphanumeric, underscores, slashes, and dots.
<img width="704" alt="image" src="https://user-images.githubusercontent.com/13956642/208638316-2fd43674-2133-4f25-a073-04c990a1fa7c.png">


Added normalization and associated unit tests since [b04143c](https://github.com/python-discord/bot/pull/2326/commits/b04143ca971e2f272ab78da808a63b5fd700ab68)

Reply:
Changed to only truncate on 85 characters. Will skip further files if less than 3 characters left in the limit. [2b4c85e](https://github.com/python-discord/bot/pull/2326/commits/2b4c85e947a73295c72161190302d960597420be)